docs/guides/blocks/sync-tables/dynamic.md

```diff
@@ -1,7 +1,428 @@
 ---
-title: Dynamic schemas
+title: Dynamic sync tables
 ---
 
 # Creating sync tables with dynamic schemas
 
-TODO
+Sync tables are designed to bring records from an external data source into Coda, but sometimes those records don't have a predefined structure. For example, a task tracking application may allow each project to define it's own set of custom fields. To bring this type of data into Coda you can create a dynamic sync table, which allows the columns in the table to adapt to the dataset that is being synced.
+
+[View Sample Code][samples]{ .md-button }
+
+
+## Using dynamic sync tables
+
+Adding a dynamic sync table to a doc is similar to adding a regular sync table, but with an additional step of selecting the specific dataset to sync from. Start by navigating to **Explore** > **Packs & import** > **{Pack name}** and clicking on the the table in the side panel. This will expand a section below the table that displays the specific datasets that the user has access to. Then drag one of these datasets onto the page.
+
+<img src="../../../../images/dynamic_sync_table_usage.gif" class="screenshot" alt="Adding a dynamic sync table to a doc">
+
+
+## Creating a dynamic sync table
+
+Creating a dynamic sync table can be a bit of an involved process, and it requires an in-depth understanding of the target API and [sync table schemas][schemas_sync_table]. The sections below cover the components of a dynamic sync table definition.
+
+
+### Define the table
+
+The dynamic sync table is defined using the [`addDynamicSyncTable()`][addDynamicSyncTable] method. The `name` is user visible and should follow the [naming conventions][sync_tables_naming] of regular sync tables. The `description` is not required but recommended.
+
+```ts
+pack.addDynamicSyncTable({
+  name: "Tasks",
+  description: "The tasks in the selected project.",
+  listDynamicUrls: async function (context) {
+    // Return the datasets the user has access to.
+    // ...
+  },
+  getName: async function (context) {
+    // Return the name of the selected dataset.
+    // ...
+  },
+  getDisplayUrl: async function (context) {
+    // Return a display URL for the selected dataset.
+    // ...
+  },
+  getSchema: async function (context) {
+    // Return a schema for the selected dataset.
+    // ...
+  },
+  formula: {
+    // Sync the rows from the dataset.
+    // ...
+  },
+});
+```
+
+More information the the various components of the dynamic sync table are described in the sections below.
+
+!!! bug
```

Comment:
Ah, this is still a bug? Does it work to override the `connectionRequirement` if you don't want to use authentication?

Reply:
Ya. Even with `connectionRequirement` of `None` you still get a **Connect** button that crashes the doc when you click it. I list the related bugs in the comment below.